spec/index.html

```diff
@@ -492,9 +492,13 @@ <h3>IRIs</h3>
       of [[!RFC3987]]. Further normalization MUST NOT be performed when
       comparing IRIs for equality.</p>
 
-      <p class="issue" data-number="4">
-        [[RFC3987]] never made it past the Proposed Standard phase
-        and should not be normatively cited.
+      <p class="issue" data-number="15">
+        A future version of this specification may use [[?URL]],
+        as [[?RFC3987]] is an abandoned specification.
+        This would include retaining most of the `IRI`-based nomenclature, but
+        defining in terms of
+        <a data-cite="URL#absolute-url-with-fragment-string">absolute-URL with fragment string</a>
+        and <a data-cite="?URL#relative-url-string">relative-URL string</a>.
       </p>
```

Comment:
It seems this is all obviated by [this comment on #15](https://github.com/w3c/rdf-concepts/issues/15#issuecomment-1436613178)